core/modules/tests/SConscript

```diff
@@ -2,4 +2,23 @@
 Import('env')
 Import('standardModule')
 
-standardModule(env, unit_tests="")
+import os.path
+
+# Harvest special binary products - files starting with the package's name
+# followed by underscore:
+#
+#   tests_<something>.cc
+#
+# Then make a Scons target for each of those to be returned by the calling
+# SConscript. Also add the source file into a list of sources to be excluded
+# from the shared library product.
+
+bin_cc_targets = {}
+path = "."
+for f in env.Glob(os.path.join(path, "tests_*.cc"), source=True, strings=True):
+    bin_cc_targets[f] = ["qserv_common","util","protobuf","log","log4cxx"]
```

Comment:
Don't overdo it. Normally all "test*.cc" files are (unit) tests and we don't want their binaries installed so don't try to make _all_ test_*.cc into true binaries. I'd probably give it different name, not starting with "test".

I believe you can also write:

    bin_cc_targets[f] = "qserv_common util protobuf log log4cxx"

Reply:
Perhaps we should come up with some naming convention for the binaries. Something like: qserv-<module>-<whatever>.cc. For instance my only app module tests should be renamed from "tests_protobuf_version.cc" into "qserv-tests-protobuf.cc". And I agree about avoiding using "test*" in apps names". I just wanted to see if my implementation of standardModule would work. And thanks for the advice to replace a list of dependencies with a string. I'll give it a try.